### PR TITLE
Added api for Vote-System

### PR DESCRIPTION
**Pull request by FlorianWip**
self explained ^^

### CODE REVIEW

File: src/main/java/eu/smashmc/api/achievements/AchievementEntity.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class AchievementEntity {
```
Comment (Sneakometer) [inline src/main/java/eu/smashmc/api/achievements/AchievementEntity.java]: Why???

File: src/main/java/eu/smashmc/api/achievements/Achievements.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package eu.smashmc.api.achievements;
+
+import eu.smashmc.api.Environment;
+import eu.smashmc.api.SmashComponent;
+
+import java.util.Set;
+
+@SmashComponent({Environment.BUKKIT, Environment.BUNGEECORD})
+public interface Achievements {
```
Comment (Sneakometer) [inline src/main/java/eu/smashmc/api/achievements/Achievements.java]: ?

File: src/main/java/eu/smashmc/api/vote/VoteEntry.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+     * @return player's name
+     */
+    private String playerName;
+
+    /**
+     * Get timestamp when the vote has been sent.
+     *
+     * @return timestamp
+     */
+    private String timestamp;
```
Comment (Sneakometer) [inline src/main/java/eu/smashmc/api/vote/VoteEntry.java]: The format is not clear. Please either change this to a java standard class (e.g. LocalDateTime) or tell the format in the documentation.

File: src/main/java/eu/smashmc/api/vote/VoteEntry.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+    /**
+     * Get the name on the site, where the player has benn vote.
```
Comment (Sneakometer) [inline src/main/java/eu/smashmc/api/vote/VoteEntry.java]: :pencil2: benn

File: src/main/java/eu/smashmc/api/vote/VoteEntry.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+     *
+     * @return the site name
+     */
+    private String serviceName;
+
+    /**
+     * Get the address from which location the player votes
+     * NOTE: Some vote sites does not send the complete address, they send for example x.x.x.0
```
Comment (Sneakometer) [inline src/main/java/eu/smashmc/api/vote/VoteEntry.java]: :pencil2: grammer 😝

**Review by Sneakometer — COMMENTED** [review]
Please resolve

File: src/main/java/eu/smashmc/api/achievements/AchievementEntity.java (hunk 1 above)
Comment (FlorianWip) [reply src/main/java/eu/smashmc/api/achievements/AchievementEntity.java]: Removed.

File: src/main/java/eu/smashmc/api/achievements/Achievements.java (hunk 2 above)
Comment (FlorianWip) [reply src/main/java/eu/smashmc/api/achievements/Achievements.java]: Removed

File: src/main/java/eu/smashmc/api/vote/VoteEntry.java (hunk 3 above)
Comment (FlorianWip) [reply src/main/java/eu/smashmc/api/vote/VoteEntry.java]: Changed to long

File: src/main/java/eu/smashmc/api/vote/VoteEntry.java (hunk 4 above)
Comment (FlorianWip) [reply src/main/java/eu/smashmc/api/vote/VoteEntry.java]: Edited

File: src/main/java/eu/smashmc/api/vote/PlayerVoteEvent.java [hunk 6]
```diff
@@ -0,0 +1,2 @@
+package eu.smashmc.api.vote;public class PlayerVoteEvent {
```
Comment (Sneakometer) [inline src/main/java/eu/smashmc/api/vote/PlayerVoteEvent.java]: What happened here?

**Review by Sneakometer — CHANGES_REQUESTED** [review]

File: src/main/java/eu/smashmc/api/vote/PlayerVoteEvent.java (hunk 6 above)
Comment (FlorianWip) [reply src/main/java/eu/smashmc/api/vote/PlayerVoteEvent.java]: Error on push/commit, fixed

Comment (FlorianWip) [reply src/main/java/eu/smashmc/api/vote/PlayerVoteEvent.java]: Fixed

**Review by Sneakometer — APPROVED** [review]

**Pull request merged**